catalog/ui/search-ui/standard/src/main/webapp/css/style.css

```diff
@@ -103,20 +103,20 @@
     cursor: pointer;
 }
 
-#layerPickers table.scrollable {
+.layerPickers table.scrollable {
```

Comment:
Previously style.css only contained CSS related to the header and footer and will someday be shared between projects in the future.  All of the layer picker CSS should be moved to searchPage.css or added to LESS.